base_kanban_stage/models/base_kanban_abstract.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import api, fields, models
+
+
+class BaseKanbanAbstract(models.AbstractModel):
+    '''Inherit from this class to add support for Kanban stages to your model.
+    All public properties are preceded with kanban_ in order to isolate from
+    child models, with the exception of stage_id, which is a required field in
+    the Kanban widget and must be defined as such.'''
```

Comment:
[PEP257](https://www.python.org/dev/peps/pep-0257/) prescribes: `For consistency, always use """triple double quotes"""`